kroxylicious-systemtests/pom.xml

```diff
@@ -149,6 +149,12 @@
             <groupId>io.kroxylicious</groupId>
             <artifactId>kroxylicious-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-operator</artifactId>
```

Comment:
I wonder if we should, in future, pull out an operator-api jar with just the CRs in it. Feel like we've discussed this before.